frontend/src/pages/order/order.tsx

```diff
@@ -0,0 +1,90 @@
+import { type Libraries, LoadScript } from '@react-google-maps/api';
+
+import { TowTruckCard } from '~/libs/components/components.js';
+import { Map } from '~/libs/components/map/map.js';
+import { useAppDispatch, useCallback, useState } from '~/libs/hooks/hooks.js';
+import { config } from '~/libs/packages/config/config.js';
+import { type OrderCreateRequestDto } from '~/packages/orders/orders.js';
+import { actions as orderActions } from '~/slices/orders/order.js';
+
+import { OrderForm } from './libs/components/order-form.js';
+import styles from './styles.module.scss';
+
+const libraries: Libraries = ['places'];
+
+//  mock
+const driverId = 1;
+
+const Order: React.FC = () => {
+  const [location, setLocation] = useState<google.maps.LatLngLiteral>({
+    // mock
+    lat: 50.4547,
+    lng: 30.5238,
+  });
+  const [destination, setDestination] = useState<google.maps.LatLngLiteral>();
+
+  const dispatch = useAppDispatch();
+
+  const handleSubmit = useCallback(
+    (payload: OrderCreateRequestDto) => {
+      void dispatch(orderActions.createOrder({ ...payload, driverId }))
+        .unwrap()
+        .then(() => {
+          // navigate user here
+        });
```

Comment:
why do you need unwrap here? can't u write just 
 dispatch(orderActions.createOrder({ ...payload, driverId }));